### PR TITLE
Added electron-simple-ipc

### PR DESCRIPTION
**Pull request by MrBlenny**
Not sure if this is of any interest...

In my fork I added [electron-simple-ipc](https://github.com/MrBlenny/electron-simple-ipc). This is a wrapper for the electron's IPC system. It both standardises the API and JSON.stringifies any objects. This makes transferring large arrays/objects much faster (about 20x faster for a 300 entry array I was testing with).

In additional, other potential improvement could be [electron-simple-rpc](https://github.com/MrBlenny/electron-simple-rpc). This is similar to your alias system but allows data/promises to be easily resolved across threads.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-dist/
```
Comment (hardchor) [inline .gitignore]: What's the reason behind this? I'd rather not commit compiled code.

Comment (MrBlenny) [inline .gitignore]: Yeh, that should be removed. Its only there so I can use the code in one of my projects without publishing to npm

**Comment by MrBlenny** [comment]
Might get round to writing some tests for it soon. Will circle back when/if it gets done.

**Comment by hardchor** [comment]
Awesome, let me know how you get on!

**Comment by hardchor** [comment]
@MrBlenny looks like you abandoned your PR ;) feel free to re-open once you've added some scripts and your lib is ready for production 😄

**Pull request closed without merging**